org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/text/correction/SourceAssistProcessor.java

```diff
@@ -165,6 +168,27 @@ public List<Either<Command, CodeAction>> getSourceActionCommands(CodeActionParam
 			AccessorField[] getters = GenerateGetterSetterOperation.getUnimplementedAccessors(type, AccessorKind.GETTER);
 			AccessorField[] setters = GenerateGetterSetterOperation.getUnimplementedAccessors(type, AccessorKind.SETTER);
 
+			if (isInFieldDeclaration) {

```

Comment:
The logic for getter and setter is too long, it's not read-friendly to put it directly in current method `getSourceActionCommands`. Can you split it to a new function to deal with getter and setter code actions?

Reply:
Extract a separate method `addGenerateAccessorsSourceActionCommand()` via https://github.com/eclipse/eclipse.jdt.ls/pull/2092/commits/a491314265ede633ad45332d2ef4b0608d189c85